src/scripts/libraryMenu.js

```diff
@@ -976,8 +978,10 @@ import 'flexStyles';
         updateLibraryNavLinks(page);
     });
 
-    events.on(window.connectionManager, 'localusersignedin', function (e, user) {
-        const currentApiClient = window.connectionManager.getApiClient(user.ServerId);
+    renderHeader();
```

Comment:
It does not seem like this function call happened before. Any idea why it is needed now?

Reply:
It was here before but was moved at the end by series of PRs: #1871 -> #1909 -> #1949
Only one of them is needed.

Reply:
I marked this as unresolved because one of the `renderHeader` calls still needs to be removed.

Reply:
removed call to renderHeader in this specific line. Comment again if that wasn't intended.